webpack.config.js

```diff
@@ -28,6 +28,7 @@ const config = {
   mode: isProduction ? "production" : "development",
   entry: {
     app: [
+      "@babel/polyfill",
```

Comment:
Not sure if this is the best place to put this (could be in `index.js` as well)

Reply:
I think it's a good place for such things as well.